dattri/scripts/cifar2_resnet9_benchmark.py

```diff
@@ -0,0 +1,365 @@
+"""This script shows how to use dattri algorithms for benchmarking CIFAR-2."""
```

Comment:
this should be convenient for users to run benchmark experiments using their own data/model?
now it is customized for cifar2 but it can be generalized to different exps just like the retrain script.
What is the reason you want to remove this?